02/2_header_and_source_file/wallet.c

```diff
@@ -0,0 +1,20 @@
+#include "wallet.h"
+
+int g_dollars = 0;
```

Comment:
only extern is allowed in header file.

global variables go to .c file

Reply:
https://stackoverflow.com/questions/8108634/global-variables-in-header-file/8108758

especially 3rd naswer